lang/java/reef-webserver/src/test/java/org/apache/reef/webserver/TestAvroSerializerForHttp.java

```diff
@@ -162,6 +163,11 @@ public int getNumberOfCores() {
     public String getRuntimeName() {
       return "Local";
     }
+
+    @Override
+    public Set<String> getNodeLabelExpressions() {
+      return null;
```

Comment:
YARN returns `null` when there is no label on a node. I think we'd better to return `null` for consistency.